app/models/educator.rb

```diff
@@ -13,6 +13,7 @@ class Educator < ActiveRecord::Base
   has_many    :interventions
   has_many    :progress_notes, through: :interventions
   has_many    :student_notes
+  has_many    :services
```

Comment:
This will have to be updated to use a different column if we revise the `services` table to store both which educator _recorded_ the service and which educator is _providing_ the service.  Since we don't have a need to query in either way at the moment, you could also just remove the association since they're easy to add when we do need them. :)